param/__init__.py

```diff
@@ -1567,14 +1567,16 @@ def _validate(self, val):
     def serialize(cls, value):
         if value is None:
             return 'null'
-        return value.to_dict('records')
+        import json
```

Comment:
As `json` is in the standard library, I would just import it at the top of the file instead of inlining it.

Reply:
Agreed, done